lib/twillo_text.rb

```diff
@@ -0,0 +1,24 @@
+require 'twilio-ruby'
+require 'dotenv/load'
+require 'rubygems'
+
+class SendMessage
+  def initialize
+    @account_sid = ENV['TWILIO_ACCOUNT_SID']
+    @auth_token = ENV['TWILIO_AUTH_TOKEN']
+    @recipient_phone = ENV['Recipient_phone']
+    @twilio_number = ENV['Twilio_number']
+  end
+
+  def send_customer_message
+    @client = Twilio::REST::Client.new @account_sid, @auth_token
```

Comment:
Again, creating a new instance of the twilio client from within a method creates a hard dependency. This time it's even more problematic as you wont be able to test your code without sending yourself a text message 🙂 

You can use dependency injection again here, although the process of doubling and stubbing required to get everything working correctly is a bit more complex this time.